packages/twenty-server/src/engine/metadata-modules/index-metadata/index-metadata.service.ts

```diff
@@ -98,6 +98,34 @@ export class IndexMetadataService {
     );
   }
 
+  async deleteIndexMetadata(
```

Comment:
In my case the index is deleted by the relation deletion. 
But it could be misleading for future users that deleteIndexMetadata does not actually deletes the index in db; although in most cases I expect it should be deleted by the original action (delete or rename a column, etc.).